bittensor/_neuron/text/advanced_server/run.py

```diff
@@ -173,14 +186,15 @@ def time_check():
         else: 
             return False
```

Comment:
Don't think we need an else here, can just return that if it fails.

Reply:
Fixed

Reply:
Fixed